src/deprecated/group.lean

```diff
@@ -269,7 +269,7 @@ end
 end ring_hom
 
 /-- Inversion is a group homomorphism if the group is commutative. -/
-@[instance, to_additive]
+@[instance, to_additive neg.is_add_group_hom "Negation is an add_group homomorphism if the add_group is commutative."]
```

Comment:
The style linter didn't complain since there are other long lines in this file.
```suggestion
@[instance, to_additive neg.is_add_group_hom 
"Negation is an `add_group` homomorphism if the `add_group` is commutative."]
```